db/migrate/20230925222043_create_tasks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTasks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :tasks do |t|
+      t.string :name
+      t.text :description
+      t.string :status
+      t.datetime :deadline
```

Comment:
When we add a date or datetime field in db, we prefer name it with `at` postfix

```suggestion
      t.datetime :deadline_at
```